var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -150,6 +160,16 @@ def cmake_args(self):
             '-DZLIB_DIR=%s' % spec['zlib'].prefix
         ])
 
+        # Set recommended flags for maximum (matrix-free) performance, see
+        # https://groups.google.com/forum/?fromgroups#!topic/dealii/3Yjy8CBIrgU
+        cxx_flags.extend([
+            '-march=native',
```

Comment:
right! will keep it for `clang`, `gcc`, `intel`. Those should be ok, as far as I know.